File: app/src/main/java/ai/kortnevdmitriy/msafiri/activities/DealOffers.java
package ai.kortnevdmitriy.msafiri.activities;

import android.os.Bundle;
import android.view.View;

import com.jetradar.desertplaceholder.DesertPlaceholder;

import ai.kortnevdmitriy.msafiri.R;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

public class DealOffers extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_deal_offers);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        DesertPlaceholder desertPlaceholder = findViewById(R.id.placeholder);
        desertPlaceholder.setOnButtonClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // do stuff
            }
        });
    }

}

File: app/src/main/java/ai/kortnevdmitriy/msafiri/activities/DealOffers.kt
package ai.kortnevdmitriy.msafiri.activities

import ai.kortnevdmitriy.msafiri.R
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.jetradar.desertplaceholder.DesertPlaceholder

class DealOffers : AppCompatActivity() {
	
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_deal_offers)
		val toolbar = findViewById<Toolbar>(R.id.toolbar)
		setSupportActionBar(toolbar)
		supportActionBar?.setDisplayHomeAsUpEnabled(true)
		
		val desertPlaceholder = findViewById<DesertPlaceholder>(R.id.placeholder)
		desertPlaceholder.setOnButtonClickListener {
			// do stuff
		}
	}
	
}
